resources/script.js

```diff
@@ -5,6 +5,23 @@ function currentPosition() {
   return parseInt(document.querySelector('.slide:not(.hidden)').id.slice(6));
 }
 
+/**
+ * If passed to navigate, will move to the first slide, independent of the
+ * current location, rather than a relative amount.
+ */
+var FIRST_SLIDE = -9999999;
+
+/**
+ * If passed to navigate, will move to the last slide, independent of the
+ * current location, rather than a relative amount.
+ */
+var LAST_SLIDE = 9999999;
+
+/**
+ * Whether to allow looping from the last back to the first or backwards from
+ * the first to the last.
+ */
+var ALLOW_LOOPING = true;
 
 /**
  * Navigates forward n pages
```

Comment:
Was going to ask why not just use 1 for `FIRST_SLIDE` but, ack!, this method is weird. I like your solution :)